src/transformStateWithClones.js

```diff
@@ -65,7 +65,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let cloneObject = { ...state };
+  const transformArray = [];
```

Comment:
```suggestion
  const states = [];
```
слово array не дописывается